haystack/components/converters/pdfminer.py

```diff
@@ -151,13 +151,19 @@ def run(
                 continue
             try:
                 pdf_reader = extract_pages(io.BytesIO(bytestream.data), laparams=self.layout_params)
-                document = self.__converter(pdf_reader)
+                document = self._converter(pdf_reader)
             except Exception as e:
                 logger.warning(
                     "Could not read {source} and convert it to Document, skipping. {error}", source=source, error=e
                 )
                 continue
 
+            if document.content == "":
```

Comment:
what do you think? Saw that the content is not fully empty but rather contains a ton of whitespace
```suggestion
            if document.content.strip() == "":
```

Reply:
hmm yeah I think that would be good for this check since I'd want to know as a user that my file just has white space as well

Reply:
@wochinge not sure about the checking for `None`, I don't think it's possible for the content to come back as None in this component.

Reply:
Okay, fair. Just wanted to be on the safe side as I think I have seen documents with content=None. Wouldn't want to run into an error when calling `strip` on `None`

Reply:
updated!